source/common/http/http1/codec_impl.cc

```diff
@@ -513,7 +513,7 @@ ConnectionImpl::ConnectionImpl(Network::Connection& connection, CodecStats& stat
       processing_trailers_(false), handling_upgrade_(false), reset_stream_called_(false),
       deferred_end_stream_headers_(false), dispatching_(false), max_headers_kb_(max_headers_kb),
       max_headers_count_(max_headers_count) {
-  if (Runtime::runtimeFeatureEnabled("envoy.reloadable_features.http1_use_balsa_parser")) {
+  if (codec_settings_.use_balsa_parser_) {
```

Comment:
should we stick with a global here so we can enable globally or locally?
actually if we use a Boolean instead of bool in the proto,we can check for presence and 
if not present default to flag
if present, use value

Reply:
Changing to BoolValue, thanks for the suggestion.